<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Agregar_moto</class>
 <widget class="QDialog" name="Agregar_moto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pb_agragar_moto_cancelar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gastos Reparacion</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_carretilla_si">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Si</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb_estado_reparado_moto">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Reparado</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado de l a Moto</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_estado_bueno_moto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Bueno</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Agregar Moto</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_placa_moto">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsb_moto_precio_compra">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Placa:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Marca:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio de Compra:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsb_moto_km_recorridos">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsb_moto_gastos_reparacion">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_estado_malo_moto">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Malo</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pb_agregar_moto_aceptar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_marca_moto">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_carretilla_no">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Km Recorridos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Trae Carretila para pasajero?</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_estado_reparado_carretilla">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Reparado</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_3</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado de la  carretila para pasajero</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_estado_bueno_carretilla">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Bueno</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_3</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb_estado_malo_carretilla">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Malo</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_3</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gastos Reparacion carretilla</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsb_moto_gastos_reparacion_carretilla">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsb_moto_precio_compra_carretilla">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio Compra carretilla</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_agragar_moto_cancelar</sender>
   <signal>clicked()</signal>
   <receiver>Agregar_moto</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
 </buttongroups>
</ui>
